GVMusicPlayerController/GVMusicPlayerController.m

```diff
@@ -384,13 +385,27 @@ - (void)doUpdateNowPlayingCenter {
     }
 
     MPNowPlayingInfoCenter *center = [MPNowPlayingInfoCenter defaultCenter];
+    NSTimeInterval elapsedInSeconds = self.currentPlaybackTime;
+    NSTimeInterval totalInSeconds = [[self.nowPlayingItem valueForProperty:MPMediaItemPropertyPlaybackDuration] doubleValue];
+    MPMusicPlaybackState state = self.playbackState;
+    float rate = (state == MPMusicPlaybackStatePlaying) ? self.currentPlaybackRate : 0;
+
     NSMutableDictionary *songInfo = [NSMutableDictionary dictionaryWithDictionary:@{
         MPMediaItemPropertyArtist: [self.nowPlayingItem valueForProperty:MPMediaItemPropertyArtist] ?: @"",
         MPMediaItemPropertyTitle: [self.nowPlayingItem valueForProperty:MPMediaItemPropertyTitle] ?: @"",
         MPMediaItemPropertyAlbumTitle: [self.nowPlayingItem valueForProperty:MPMediaItemPropertyAlbumTitle] ?: @"",
-        MPMediaItemPropertyPlaybackDuration: [self.nowPlayingItem valueForProperty:MPMediaItemPropertyPlaybackDuration] ?: @0
+        MPMediaItemPropertyPlaybackDuration: @(totalInSeconds),
+        MPNowPlayingInfoPropertyElapsedPlaybackTime: @(elapsedInSeconds),
+        MPNowPlayingInfoPropertyPlaybackProgress: @(elapsedInSeconds/totalInSeconds),
```

Comment:
Well, in fact there is none because of how the system behaves, further details here:
http://stackoverflow.com/questions/1449109/why-doesnt-this-crash-arent-i-dividing-by-zero-here

But it's definitely suspiciously looking operation. I've patched this in the following commit along with a symbol checking for the other properties I've added as these are officially available since SDK 10.0.

There are potentially more places that could cause troubles because of the SDK availability stuff since iOS 7.0 or 9.x so I wouldn't guarantee that this library still works correctly on iOS 5+6 like the Example deployment setting declares.